Patches/Fell Tribes/ThingDefs_Races/Bear.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- ========== Make sure the mod is there first ========== -->
+	<Operation Class="PatchOperationSequence">

```

Comment:
This is an older style of FindMod. Please use the newer versions that can be found in patches, like this one. The end of the file will also need some tweaks, as shown.

```
<Patch>

  <Operation Class="PatchOperationFindMod">
    <mods>
      <li>Fell Tribes</li>
    </mods>
    <match Class="PatchOperationSequence">
      <operations>

(your operations go here)

      </operations>
    </match>
  </Operation>
</Patch>    
```

Reply:
Done.

Reply:
Umm... It doesn't seem to be? Please swap in the new format of FindMod.